napari/layers/shapes/_shapes_models/shape.py

```diff
@@ -168,7 +168,11 @@ def dims_displayed(self):
     @property
     def bounding_box(self) -> np.ndarray:
         """(2, N) array, bounding box of the object."""
-        return self._bounding_box[:, self.dims_displayed]
+        # We add +-0.5 to handle edge width
```

Comment:
Interesting, but in [this test](https://github.com/napari/napari/pull/7332/files#diff-1048159db6d94b0a31853bf61d0955f7f303b7ce76a7b4e79a537e1097c68ef5R36) I can see that the full edge width is accounted for (if the edge width is different from 1), so how does *that* work???

Reply:
wait I just saw that you multiply below 😂 ignore me!